recipes/mir-prefer/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "miR-PREFeR" %}
+{% set version = "0.24" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}_{{ version }}.tar.gz
+  url: https://github.com/hangelwen/miR-PREFeR/archive/v0.24.tar.gz
+  md5: c78ae8bbb7daba266648da1a9d4a2f27 
+  patches:
+    - patches/bowtie-align-reads.py.patch
+    - patches/convert-mirdeep2-fasta.py.patch
+    - patches/convert-readcount-file.py.patch
+    - patches/miR_PREFeR.py.patch
+    - patches/process-reads-fasta.py.patch	
+
+build:
+  number: 0
+  skip: True # [not py27]
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python 
+    - viennarna
+    - samtools ==0.1.19
+    - bowtie ==1.2.0
+
+test:
+  commands:
+    - bowtie-align-reads.py -h 2>&1 | grep "python bowtie-align-reads.py" > /dev/null
```

Comment:
In general you don't need this /dev/null and it will see in the error logs more.